android/app/src/main/java/app/organicmaps/MwmActivity.java

```diff
@@ -1249,9 +1248,7 @@ public void onSwitchFullScreenMode()
 
   private void setFullscreen(boolean isFullscreen)
   {
-    if (RoutingController.get().isNavigating()
-        || RoutingController.get().isBuilding()
-        || RoutingController.get().isPlanning())
```

Comment:
This 'if' block was added in 2016 in commit [e66fe5b](https://github.com/organicmaps/organicmaps/commit/e66fe5b62d63094f6781aafe9025a614303a33ba). I reduced it to only `isNavigation()` condition. Any objections?

Reply:
@strump does it mean that the interface will hide when the route is planned or is built? That should not happen. Full screen is disabled on iOS in such cases.

Reply:
Reverted this change. Now Android exits fullscreen to start route planning. In route planning mode no fullscreen mode allowed.